README.md

```diff
@@ -68,7 +68,19 @@ class MyCharm(CharmBase):
 ```
 
 Every standard event in Juju may be observed that way, and you can also easily
-define your own events in your custom types.
+define your own events in your custom types. If you name your observer with the
+scheme `on_<event-name>` then you can optionally skip explicitly referencing it
```

Comment:
Might be worth using "config-changed" as an example, to show that dashes are converted to lower case

Reply:
Makes sense! Let me work on that